### ISSUE TITLE
Add Travis build for OQS-OpenSSL

### ISSUE THREAD

**Issue by christianpaquin** [open-quantum-safe/openssl#36]
Provide CI testing for the OQS fork

**Comment by christianpaquin** [comment]
@smashra, are you planning to update the scripts you circulated on the dev mailing list yesterday?

**Comment by smashra** [comment]
Yes. Now that we have test repository I’ll be checking in there.

**Comment by christianpaquin** [comment]
I tried to enable travis on OpenSSL, but I can't figure out how. I logged in travis but the open-quantum-safe projects were not listed in the list of projects I could travis-enable. It might be because I don't have the right github ownership/permissions for this project? @smashra, @dstebila; can one of you can try to turn on travis? The OpenSSL project already has a travis test suites (and also an appveyor one); so we should turn these to make sure our PRs don't break the main OpenSSL functionality. As a step 2, we can add some OQS tests.

**Comment by dstebila** [comment]
I've turned Travis on for this repository.

@smashra and I were discussing whether/how to trigger OpenSSL/OpenSSH integration tests when just a liboqs commit is made.  I noticed in the Travis settings it's possible to set a cron job to regularly build a repository.  But also we might want to add Travis target(s) in each of the liboqs master and nist .travis.yml files that runs the integration test script(s).
